### PR TITLE
WIP: Policies and procedures for sharing modules

### PR DESCRIPTION
**Pull request by bhardesty**
This is an initial stab at some policies and procedures for sharing modules, reusing them in a RH Middleware book, and making updates to a shared module. There are still some items to consider, such as how to handle modules with images.

### CODE REVIEW

**Comment by rkratky** [comment]
The high-level parts could be useful to all docs teams, WDYT?

**Comment by ncbaratta** [comment]
@rkratky we were using this as temporary bandaid while we wait for the module library. Once we have that we'll all use that.  For that reason, we were only doing this in middleware at this time, but this is all public so anyone can take what we have and use it.

File: README.md [hunk 1]
```diff
@@ -1,85 +1,75 @@
...
+    b. Add your modules to the directory that you created.
+
+    c. Add a `README.md` file and describe how the set of modules should be used.
+
+      * List any attributes that the modules use.
+      * Describe how the modules should be included. At a minimum, define the order in which the modules should be included.
+
+2. Create a PR to add the module.
+
+2. Wait. The PR will be reviewed and, once approved, merged.
```
Comment (ncbaratta) [inline README.md]:
```suggestion
3. Wait. The PR will be reviewed and, once approved, merged.
```

File: README.md [hunk 2]
```diff
@@ -1,85 +1,75 @@
...
+
+3. After the PR is merged:
```
Comment (ncbaratta) [inline README.md]:
```suggestion
4. After the PR is merged:
```

**Pull request merged**